packages/web-react/src/components/UNSTABLE_EmptyState/stories/UNSTABLE_EmptyState.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import { ButtonLink } from '../../Button';
+import { Heading } from '../../Heading';
+import { Link } from '../../Link';
+import { Text } from '../../Text';
+import { UNSTABLE_ActionLayout } from '../../UNSTABLE_ActionLayout';
+import ReadMe from '../README.md';
+import UNSTABLE_EmptyStateSection from '../UNSTABLE_EmptyStateSection';
+import { UNSTABLE_EmptyState } from '..';
```

Comment:
```suggestion
import { UNSTABLE_EmptyStateSection, UNSTABLE_EmptyState } from '..';
```

Reply:
Please, prefer importing from a component file instead of the index.